src/databroker_elasticsearch/__init__.py

```diff
@@ -8,3 +8,86 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+
+def callback_from_config(config):
+    """Construct ElasticCallback instance from a configuration dictionary.
+
+    Parameters
+    ----------
+    config : dict
+        The dictionary which contains "databroker-elasticsearch" key.
+
+    Returns
+    -------
+    ElasticCallback
+        The new instance of ElasticCallback.
+    """
+    from databroker_elasticsearch.elasticcallback import ElasticCallback
+    rv = ElasticCallback.from_config(config)
+    return rv
+
+
+def callback_from_name(name):
+    """Construct ElasticCallback from YAML configuration in standard paths.
+
+    Parameters
+    ----------
+    name : str
+        The base name of YAML configuration file to be found and loaded.
+        For example, the name "xyz-es" would be looked up as
+
+        * ``~/.config/databroker/xyz-es.yml``
```

Comment:
@stuartcampbell is this where this should be stored? Or should we put it under it's own dir?

Reply:
The locations are from databroker `lookup_config` function.
We also have `load_callback` factory which can load from anywhere.